.gitlab-ci.yml

```diff
@@ -168,7 +185,13 @@ examples-clippy-wasm:
 examples-contract-build:
   stage:                           examples
   <<:                              *docker-env
+  dependencies:
+    - build-cli
+  needs:
+    - build-cli
```

Comment:
Why does it need both? Isn't `dependencies: build-cli` enough here?

Reply:
I'll check later, but `dependencies` are set for uploading artifacts from that job. `needs` is to start right after that job succeds.

Reply:
:ok_hand: